### PR TITLE
Fix paging

### PR DESCRIPTION
**Pull request by tripox**
BBS-304

### CODE REVIEW

File: modules/primo/includes/primo.search.inc [hunk 1]
```diff
@@ -202,7 +202,7 @@ function primo_search_search(TingSearchRequest $ting_query) {
     }
   }
 
-  $page = $ting_query->getPage();
+  $page = $ting_query->getPage() - 1;
```
Comment (kasperg) [inline modules/primo/includes/primo.search.inc]: This seems odd. The documentation states that you use 1 for the first page. Subtracting 1 means that `$ting_query->getPage()` returns 2 for the first page?

File: modules/primo/src/Primo/BriefSearch/Client.php [hunk 2]
```diff
@@ -144,10 +144,11 @@ public function validateThumbnail($url) {
    * @throws \Primo\Exception\TransferException
    *   If an error occurs during the execution of the search.
    */
-  public function search(array $queryParameters, $offset, $numResults, $sort = NULL) {
+  public function search(array $queryParameters, $page, $count, $sort = NULL) {
+    $offset = ($count * $page) + 1;
```
Comment (kasperg) [inline modules/primo/src/Primo/BriefSearch/Client.php]: Using 1 for the first page means that `$offset` will be 11 for `$page = 1` and `$count = 10`. That cannot be right?

File: modules/primo/src/Primo/BriefSearch/Client.php [hunk 3]
```diff
@@ -144,10 +144,11 @@ public function validateThumbnail($url) {
    * @throws \Primo\Exception\TransferException
    *   If an error occurs during the execution of the search.
    */
-  public function search(array $queryParameters, $offset, $numResults, $sort = NULL) {
+  public function search(array $queryParameters, $page, $count, $sort = NULL) {
```
Comment (kasperg) [inline modules/primo/src/Primo/BriefSearch/Client.php]: This changes the function arguments. As I see it the function is called in multiple places but you only update one, `primo_search_search()`. Have you checked the other places?

File: modules/primo/src/Primo/BriefSearch/Client.php (hunk 2 above)
Comment (tripox) [reply modules/primo/src/Primo/BriefSearch/Client.php]: That's right:
https://developers.exlibrisgroup.com/primo/apis/webservices/xservices/search/briefsearch

See `indx` and `bulkSize`

**Pull request merged**